### ISSUE TITLE
fallback python-based .rtf extraction

### ISSUE THREAD

**Issue by deanmalmgren** [deanmalmgren/textract#88]
This is currently using the `unrtf` command line tool, but it would be nice to have a pure python extraction method as a fallback.

**Comment by pombredanne** [comment]
@deanmalmgren Have a look at https://github.com/brendonh/pyth

**Comment by deanmalmgren** [comment]
Nice find, @pombredanne; thanks for pointing this out! 

If anyone is interested in taking a crack at this, the [pdf parser](https://github.com/deanmalmgren/textract/blob/master/textract/parsers/pdf_parser.py) has a pretty good example for how to have multiple `method`s for extracting text from documents.

**Comment by jpadilla** [comment]
@deanmalmgren I'll try to get this done in a few.

**Comment by deanmalmgren** [comment]
Awesome, thanks @jpadilla!

**Comment by jpadilla** [comment]
For one thing, pyth seems to have issues with charsets on the existing RTF files in textract which will make it harder to test. Might be related to https://github.com/brendonh/pyth/pull/30

**Comment by pombredanne** [comment]
@jpadilla indeed, and I ran pyth through the test files from unrtf and submitted a ticket there: https://github.com/brendonh/pyth/issues/34 
Which is an opportunity to help make @brendonh pyth better?

**Comment by KyleKing** [comment]
Unfortunately pyth doesn't appear to be maintained and there doesn't appear to be a good unrtf Python alternative. I'm going to close this ticket while cleaning up the backlog, but I'm open to ideas on implementation and reopening this if there is!

**Comment by KyleKing** [comment]
Actually, https://github.com/joshy/striprtf looks like a good option. If that sounds good to others, I can submit a PR or I would welcome any contributions
